Index: cjdns-HEAD/node_build/make.js
===================================================================
--- cjdns-HEAD.orig/node_build/make.js	2014-04-18 10:39:55.000000000 +0400
+++ cjdns-HEAD/node_build/make.js	2014-04-18 11:44:24.730137000 +0400
@@ -211,6 +211,9 @@
         builder.config.includeDirs.push(
             BUILDDIR+'/dependencies/libuv/include/'
         );
+	builder.config.cflags.push(
+	    '-fno-stack-protector'
+	);
         Fs.exists(BUILDDIR+'/dependencies/libuv/out/Release/libuv.a', waitFor(function (exists) {
             if (exists) { return; }
             console.log("Build Libuv");
